packages/babel-helpers/src/helpers.js

```diff
@@ -425,6 +425,47 @@ helpers.inheritsLoose = defineHelper(`
   }
 `);
 
+// Based on https://github.com/WebReflection/babel-plugin-transform-builtin-classes
+helpers.wrapNativeSuper = defineHelper(`
+  var _gPO = Object.getPrototypeOf || function _gPO(o) { return o.__proto__ };
```

Comment:
this patch is explicitly incompatible with IE <= 10 where you should *not* extend native builtins

Reply:
But, if we're writing new code that extends builtins...?

Reply:
then you are responsible to make such code work as expected otherwise you should stop using JavaScript since even `Object` could be overwritten :smile:

Reply:
Gotcha, let me stop using JS on my next web app!